docs/what_makes_securedrop_unique.rst

```diff
@@ -0,0 +1,105 @@
+information than email providers or phone companies do.
+
+The source can only log into SecureDrop through the Tor Browser, which masks the
+source’s IP address to begin with, so there is no indication who the source is
+(unless they disclose it) and where they are sending information from. The Tor IP
+address, the computer and the browser type that the source is using is not logged
+either.
+
+For each source, only the time and date of each submission is logged on the
+server. When a source sends a new message the time and date of the last message
+is overwritten. This means that there won’t be a trail of metadata showing
+exactly when the source and journalist were talking.
+
+In addition, sources cannot create a custom username that could reveal information
+about them. Instead, SecureDrop automatically generates two random codenames, one
+to show to the source and another to the journalists using the system.
+
+Encrypted and air-gapped
+------------------------
+
+Communications through SecureDrop are both encrypted in transit, so messages cannot
+be easily intercepted and read while they are traversing the internet and are also
+encrypted on the server so if any attacker manages to break into the server, they
+would not be able to read past messages.
+
+In addition, the decryption key for SecureDrop submissions sits on an air-gapped
+computer (not connected to the Internet). This air-gapped computer is the only
+place SecureDrop submissions are decrypted and read so that they are much harder
+for an attacker to access.
+
+Protecting against hackers
+--------------------------
+
+A 2014 study showed that 20 of the top 25 news organization had, at one time or
+another, had their networks compromised by state sponsored hackers.
+
+Because of this threat, SecureDrop completely segments its traffic from a news
+organization’s normal network. Submissions are accessed and downloaded using the
+Tails operating system, which boots off of a USB, does not touch your computer’s
+hard drive, and routes all its internet traffic through Tor.
+
+Submissions are decrypted on an air-gapped computer also using Tails. This
+mitigates against the risk that an attacker could send malware through SecureDrop
+in an attempt to infect the news organization’s normal network as well.
+
+The SecureDrop servers also undergo significant system hardening in order to
+make it as difficult as possible for hackers to break in.
+
+So SecureDrop protects sources against networks that are already compromised and
+also protects a news organization’s normal network from attacks that could
+potentially come through SecureDrop.
+
+Free and open source software
+-----------------------------
+
+100% of SecureDrop’s code is free and open source. Not only does this mean anyone
+can install SecureDrop themselves, but the code is available online for security
+experts to test for vulnerabilities.
+
+SecureDrop has gone through at least four audits by third party penetration
```

Comment:
Remove "at least", as we should be more concrete with the number of audits we've had.